options/js/options.js

```diff
@@ -1,29 +1,64 @@
-
-var radios = document.getElementsByName('theme');
+var retrieveQuery=[];
```

Comment:
we can have better name for these variables something like `recentSearchQueries` and `recentSearchQueryUrls`. Also please can you share do you want to store something like last 5 queries or the complete history. If the complete history have you tested the UI not to be breaking if a person makes a lot of searches.

Reply:
Also, we should try to name arrays as plural it helps in cases when you want to use map, reduce, filter and forEach as you can simply go with -> `recentSearchQueries.map(query=>{ })` instead of using random names for each element of array ex - el, or something else.

Reply:
I have renamed the arrays, and in the process of doing so I realized the the 2 arrays in the popup.js and options.js files can have the same names since their scopes do not interfere, so I have done that to avoid confusion.


I have tested the UI for upto 29 history entries. It does not affect the UI in any adverse way. When the size of the List exceeds the body height, a scroll bar is provided to access the entries.

![image](https://user-images.githubusercontent.com/40874041/52587836-06111900-2e61-11e9-9d29-eba4d39e5238.png)